docs/installing.rst

```diff
@@ -21,7 +19,11 @@ Installing with get-pip.py
 --------------------------
 
 To install pip, securely download `get-pip.py
-<https://bootstrap.pypa.io/get-pip.py>`_. [2]_
+<https://bootstrap.pypa.io/get-pip.py>`_. [2]_ i.e. 
+
+::
+
+ curl https://bootstrap.pypa.io/get-pip.py -o - | python
```

Comment:
curl + pipe is [definitively a security issue](https://www.idontplaydarts.com/2016/04/detecting-curl-pipe-bash-server-side/), as server-side can detect that you are piping in many cases.